elasticsearch/participants_structured_extractor.py

```diff
@@ -0,0 +1,12 @@
+import sys
+import json
+
+def extract():
+    aliases = sys.argv[1]
+    participants_structured = json.loads(aliases)
+    filtered_indexes = list(filter(lambda x: x.startswith('participants_structured.'), participants_structured))
+    for item in filtered_indexes:
```

Comment:
*minor*
 
although not necessary here list unpacking operator can be used to make the code shorter:

separated by new line:
print(*filtered_indexes, sep="\n")

separated by space:
print(*filttered_indexes)